lib/internal/cluster/worker.js

```diff
@@ -3,8 +3,7 @@ const EventEmitter = require('events');
 const internalUtil = require('internal/util');
 const util = require('util');
 const defineProperty = Object.defineProperty;
-const suicideDeprecationMessage =
-    'worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.';
+const deprecationMessage = 'Please use worker.exitedAfterDisconnect instead.';
```

Comment:
Understood, and in general I agree that it's not ideal, but this is about finding a compromise that avoids the original problem. The reason deprecation codes were added is to provide a mechanism for *finding* the context if necessary.